Clarified Server version and accessibility .NET version (#111709)
Azure/azure-content: articles/active-directory/hybrid/connect/how-to-connect-install-prerequisites.md

#### Installation prerequisites

> - Azure AD Connect must be installed on a domain-joined Windows Server 2016 or later. We recommend the usage of domain joined Windows Server 2022. You can deploy Azure AD Connect on Windows Server 2016 but since Windows Server 2016 is in extended support, you may require [a paid support program](/lifecycle/policies/fixed#extended-support) if you require support for this configuration.

Reviewer: ```suggestion
- Azure AD Connect must be installed on a domain-joined Windows Server 2016 or later. We recommend using domain-joined Windows Server 2022. You can deploy Azure AD Connect on Windows Server 2016 but since Windows Server 2016 is in extended support, you may require [a paid support program](/lifecycle/policies/fixed#extended-support) if you require support for this configuration.
```
